client/src/pages/SignUp/SignUp.tsx

```diff
@@ -38,24 +37,23 @@ export default function Register(): JSX.Element {
 
   return (
     <Grid container component="main" className={classes.root}>
-      <Grid item xs={12} sm={8} md={7} elevation={6} component={Paper} square>
+      <Grid item xs={12} elevation={6} component={Paper} square>
         <Box
           display="flex"
           justifyContent="space-between"
           alignItems="flex-start"
           flexDirection="column"
           className={classes.authWrapper}
         >
-          <AuthHeader linkTo="/login" asideText="Already have an account?" btnText="Login" />
-          <Box width="100%" maxWidth={450} p={3} alignSelf="center">
-            <Grid container>
-              <Grid item xs>
-                <Typography className={classes.welcome} component="h1" variant="h5">
-                  Create an account
-                </Typography>
-              </Grid>
-            </Grid>
+          {/* <AuthHeader linkTo="/login" asideText="Already have an account?" btnText="Login" /> */}
```

Comment:
I see why you commented this out, could we actually maybe repurpose this to be a AuthFooter and have it used both in Sign up and login forms?

Reply:
If you don't want to do that, just remove it from the code and delete the component from the project.